test/integration/targets/tags/runme.sh

```diff
@@ -47,3 +47,16 @@ export LC_ALL=en_US.UTF-8
 # Run templated tags
 [ "$("${COMMAND[@]}" --tags tag3 | grep -F Task_with | xargs)" = \
 "Task_with_always_tag TAGS: [always] Task_with_templated_tags TAGS: [tag3]" ]
+
```

Comment:
Tests are failing because this file was made non-executable.

Reply:
thanks for the hint